client/src/components/ItemDisplay.js

```diff
@@ -56,10 +57,20 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+// Adds a comma once every 3 digits right to left
+const addCommasToDollars = (dollars) => {
+  let dollarsArray = String(dollars).split("");
```

Comment:
You use String() on line 69 when defining dollars, so it shouldn't be necessary here